### ISSUE TITLE
Output order different than what is in tests

### ISSUE THREAD

**Issue by bruchar1** [pkgconf/pkgconf#306]
I compiled pkgconf on Windows. I tried to reproduce the test cases in `basic.sh`. Some cases give me a different output order than in the test. For instance:

```
$ build\pkgconf.exe --with-path=\env\pkgconf\tests\lib1\ --libs intermediary-1 intermediary-2                                                                                                                                  
-lintermediary-2 -lintermediary-1 -lfoo -lbar -lbaz
```
while the test says the output should be : `-lintermediary-1 -lintermediary-2 -lfoo -lbar -lbaz\n`

Do you have any idea why this order differs? Does it matter?

**Comment by kaniini** [comment]
Probably fixed by 5d0e2f50923aff3ce4778bc3ab39452fc2daec5f, it turns out that qsort is not stable on Windows.

**Comment by bruchar1** [comment]
I confirm it fixes the issue. Thank you!
